ckcp/README.MD

```diff
@@ -32,6 +32,7 @@ Before you execute the script,
 3. You need to install [oc](https://docs.openshift.com/container-platform/4.9/cli_reference/openshift_cli/getting-started-cli.html)
 4. You need to install [argocd](https://argo-cd.readthedocs.io/en/stable/cli_installation/).
 5. You need to install [yq](https://mikefarah.gitbook.io/yq/#install)
```

Comment:
yq does not seem to be used anywhere anymore, is it?

Reply:
not sure, it's a portable command-line YAML, JSON and XML processor.  It will be easier to process yaml in our script.